code/tim/HTML+CSS/lab01_Bio/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Brendan Fraser</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+    
+    <div id="bio_info">
+    <h2>Brendan Fraser</h2>
+    <h3>Brendan James Fraser was born in Indianapolis, Indiana, to Canadian parents Carol Mary (Genereux), a sales counselor, and Peter Fraser, a journalist and travel executive. He is of Irish, Scottish, German, Czech, and French-Canadian ancestry. As his parents frequently moved, Brendan can claim affinity with Ottawa, Indianapolis, Detroit, Seattle, London and Rome. His early exposure to theatre, particularly in London, led him to Seattle's Cornish Institute.</h3>
+        
+    <h3>After graduation he found a minor role as Sailor #1 in River Phoenix's Dogfight (1991), then somewhat more substantial roles in Encino Man (1992) and School Ties (1992). He expresses a preference for playing "fish out of water" men. Five more years of supporting work led finally to the title role in George of the Jungle (1997), a role which fully utilized his charm and beefy good looks, as well as offering him a chance to show off his comic talents. He describes this role as the one which dramatically altered his career.</h3>    
+    
+    <ul type="square" id="has lived">He has lived in 
```

Comment:
These line items are also not centered. Should be a quick fix. Let me know if you need help.